### PR TITLE
Fix #371: Add custom page titles.

### LINKED ISSUE #371: HTML website: Add page title
Currently, for most (or all) pages on release-monitoring.org, the title simply says "Anitya" in browser tab, browser header bar and history (Firefox 49.0.2, but that should be consistent on all major browsers). This is due to the `<title>Anitya</title>` tag.
It would be nice to have a more meaningful title like `[project name] project on Anitya` or `[project name] project on release monitoring`. Same for all the other pages you might be ending up on: "Edit", "Flag", "Add Project", "About", "Distros", …

### PR DESCRIPTION
**Pull request by souravbadami**
This commit is to discuss the pages to consider for custom page titles.

### CODE REVIEW

**Comment by souravbadami** [comment]
Added custom page titles. Here's the list of pages included:
- api.html
- distro_add.html
- distro_delete.html
- distro_edit.html
- distros.html
- docs.html
- flags.html
- index.html
- login.html
- logs.html
- mapping.html
- master.html
- project.html
- project_delete.html
- project_flag.html
- project_new.html
- projects.html
- search.html
- updates.html

File: anitya/templates/distros.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {% extends "master.html" %}
 
+{% block title %}Distributions Participaring &mdash; Anitya{% endblock %}
```
Comment (jeremycline) [inline anitya/templates/distros.html]: There's a typo here, it should be ``Participating``

File: anitya/templates/api.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {% extends "master.html" %}
 
-{% block title %} API | Anitya {% endblock %}
+{% block title %} API &mdash; Anitya {% endblock %}
```
Comment (jeremycline) [inline anitya/templates/api.html]: This is personal preference, but can we use a narrower separating character here? Tabs can get pretty narrow and space is a premium. I got 2 more characters on my Firefox tab using the middle dot character, ``·``.

Comment (souravbadami) [inline anitya/templates/api.html]: I read an article, **Choosing the best title separator** (https://perishablepress.com/choosing-the-best-title-separators/). There's no specific rule to select a separator over another. It just clears the mess and makes the title readable. I think, we can go with a shorter separator, like: `·` or `-`. What do you say ?

Comment (jeremycline) [inline anitya/templates/api.html]: I'm okay with either one, so unless someone else has very strong feelings about title separators, I'd say pick whatever you like most.

Comment (souravbadami) [inline anitya/templates/api.html]: Okay, I'll make the necessary changes.

File: anitya/templates/distros.html (hunk 1 above)
Comment (souravbadami) [inline anitya/templates/distros.html]: Done.

**Pull request merged**